source/infrastructure/jest.config.ts

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   "roots": [
-    "<rootDir>/test"
+    "<rootDir>/tests"
```

Comment:
the .gitignore was filtering it out, options were to either make the config .ts or manually exclude it from the ignore. Either approach would work but .ts is a natively supported config type for jest so went for the approach that didn't need me to mess with the .gitignore

Reply:
that said, I did endeup having to manually exclude it from the .tsconfig instead, so ended up not saving anything. I dont think it matters if this is .ts or .js, no actual difference in behavior

Reply:
For ASR I added `!jest.config.js` to my .gitignore.
Looks like jest is totally fine with this: https://jestjs.io/docs/configuration

Reply:
Are you saying you think it's better if this was in a jest.config.js file instead of .ts? or do we not really care either way (Jest supports either natively)

Reply:
I wasn't sure if it mattered. I confirmed that it doesn't matter. I say keep it the way you have it here, as a `.ts` file.